src/carma-streets/plot_message_frequencies.py

```diff
@@ -23,42 +25,79 @@ def plot_message_frequencies(csv_dir: Path, plots_dir: Path, simulation: bool =
         # Read CSV data
         message_data = dict([])
         time_sync_data = None
-        for csv_file in csv_dir.glob("*.csv"):
-            print(f'Reading csv file {csv_file.name} ...')
+        is_all_message_types = message_type == "all"
+        file_pattern = "*.csv" if is_all_message_types else str(message_type) + ".csv"
```

Comment:
I don't understand this syntax. Could you explain it? Is this how it works
```python3
if is_all_message_types:
  return "*.csv
else:
   str(message_type) + ".csv // array of string message types
```

Reply:
-` is_all_message_types = message_type == "all"` will assign the `is_all_message_types `to true if the `message_type `is `"all"`, otherwise it is `false`.

- The above statement will assign `file_pattern `variable to value `"*csv"` if `is_all_message_types== true`, otherwise the file_pattern value is ` str(message_type) + ".csv" `. E.G, `message_type=="sdsm"`, then file_pattern is assigned `"sdsm.csv"`